cl/corpus_importer/tests.py

```diff
@@ -791,3 +811,20 @@ def test_no_volume_citation(self):
         )
         self.read_json_func.return_value = case_law
         self.assertSuccessfulParse(1)
+
+    def test_failing_name(self):
+        """
+        Check if there is an overlap between two similar case names.
+        """
```

Comment:
Probably could use a more descriptive name.  

Test case name winnowing comparison ... or 

test_removal United States... something something

Reply:
for the function name or the description, or both?

Reply:
both.

Reply:
What do you think about this

```
    def test_case_name_winnowing_comparison(self):
        """
        Test removing "United States" from case names and check if there is an
        overlap between two case names.
        """
        case_name_full = "UNITED STATES of America, Plaintiff-Appellee, " \
                         "v. Wayne VINSON, Defendant-Appellant "
        case_name_abbreviation = "United States v. Vinson"
        harvard_case = f"{case_name_full} {case_name_abbreviation}"

        case_name_cl = "United States v. Frank Esquivel"
        overlap = winnow_case_name(case_name_cl) & winnow_case_name(
            harvard_case)
        self.assertEqual(len(overlap), 0)
```